src/generator/backend/fold.js

```diff
@@ -47,10 +47,18 @@ function zeroFill(num) {
   }
   return '0' + num.toString();
 }
-function foldByTrack(sessions, speakers) {
+function returnTrackColor(trackInfo,id){
```

Comment:
Fix the space issues from codacy. They are valid warnings.

On Sat 18 Jun, 2016 03:25 Codacy Bot, <notifications@github.com> wrote:

> In src/generator/backend/fold.js
> <https://github.com/fossasia/open-event-webapp/pull/303#discussion_r67580892>
> :
>
> > @@ -47,10 +47,18 @@ function zeroFill(num) {
> >    }
> >    return '0' + num.toString();
> >  }
> > -function foldByTrack(sessions, speakers) {
> > +function returnTrackColor(trackInfo,id){
>
> [image: Codacy]
> <https://camo.githubusercontent.com/37ddd55708a38212f4e9de4e3eac98358fefe597/68747470733a2f2f7777772e636f646163792e636f6d2f6173736574732f696d616765732f66617669636f6e2e706e67>
> Issue found: Missing space before opening brace.
> <https://www.codacy.com/app/triplez/open-event-webapp/file/2787446754/issues/source?bid=3233857&fileBranchId=3368529#l50>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fossasia/open-event-webapp/pull/303/files/aec13e084d7827cae57440e7dab6b9f97702befc#r67580892>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQ_yq2Wf40kFrAb4kfqkrPENoeg_aHHks5qMxfAgaJpZM4I4wmX>
> .
>